wagtailio/settings/production.py

```diff
@@ -28,6 +28,10 @@
 
 FB_APP_ID = 710333029076622
 
+VERCEL_DEPLOY_HOOK_URL = env.get("VERCEL_DEPLOY_HOOK_URL", None)
+VERCEL_DEPLOY_REQUEST_TIMEOUT = env.get("VERCEL_DEPLOY_REQUEST_TIMEOUT", 0.001)
```

Comment:
This is an interesting idea!

If we don't wait longer than 1ms for a response, we probably don't need to run it in a thread! But should we catch the timeout error though?

Reply:
Do I increase the timeout or remove the threads?

Reply:
It might be fine to leave it something really small (like 1 second) as long as we catch the timeout error. We don't actually do anything with the response so I don't think we need to wait for it.